AmazonChimeSDK/AmazonChimeSDK/internal/utils/DeviceUtils.swift

```diff
@@ -60,9 +61,18 @@ import UIKit
         appInfo.deviceMake = "apple"
         appInfo.clientSource = "amazon-chime-sdk"
         appInfo.chimeSdkVersion = Versioning.sdkVersion()
+        appInfo.clientUtcOffset = getClientUtcOffset()
         return appInfo
     }
 
+    static private func getClientUtcOffset() -> String {
```

Comment:
Do we except unit tests for this?

Reply:
Actually there were no unit tests for this file. I have moved this function to DateUtils and added unit tests for it.